modules/maximum-point/maximum-point.ts

```diff
@@ -0,0 +1,264 @@
+          break;
+        }
+        case SUB_COMMAND_MY_POINTS: {
+          await this.showMyPoints(interaction);
+          break;
+        }
+        case SUB_COMMAND_HELP: {
+          await interaction.reply({
+            content: this.help(),
+            ephemeral: true,
+          });
+          break;
+        }
+        case SUB_COMMAND_INFO: {
+          await interaction.reply({
+            content: this.info(),
+            ephemeral: true,
+          });
+          break;
+        }
+      }
+    });
+
+    this.client.on("messageReactionAdd", async (reaction, user) => {
+      await this.handleReactionAdd(reaction, user);
+    });
+
+    this.client.on("messageReactionRemove", async (reaction, user) => {
+      await this.handleReactionRemove(reaction, user);
+    });
+  }
+
+  private async handleReactionAdd(
+    reaction: MessageReaction | PartialMessageReaction,
+    user: User | PartialUser
+  ) {
+    if (user.bot) return;
+
+    if (reaction.partial) {
+      try {
+        await reaction.fetch();
+      } catch (error) {
+        console.error("Failed to fetch reaction:", error);
+        return;
+      }
+    }
+
+    const emoji = reaction.emoji.name || reaction.emoji.id || "";
+
+    if (!this.pointManager.isTargetEmoji(emoji)) return;
+
+    const targetUser = reaction.message.author;
+    if (!targetUser || targetUser.bot) return;
+
+    if (targetUser.id === user.id) return;
+
+    this.pointManager.addPoint(
+      targetUser.id,
+      targetUser.username || targetUser.id,
+      1
+    );
+
+    console.log(
+      `Added 1 point to ${targetUser.username} (${targetUser.id}) by ${user.username}`
+    );
+  }
+
+  private async handleReactionRemove(
+    reaction: MessageReaction | PartialMessageReaction,
+    user: User | PartialUser
+  ) {
+    if (user.bot) return;
+
+    if (reaction.partial) {
+      try {
+        await reaction.fetch();
+      } catch (error) {
+        console.error("Failed to fetch reaction:", error);
+        return;
+      }
+    }
+
+    const emoji = reaction.emoji.name || reaction.emoji.id || "";
+
+    if (!this.pointManager.isTargetEmoji(emoji)) return;
+
+    const targetUser = reaction.message.author;
+    if (!targetUser || targetUser.bot) return;
+
+    if (targetUser.id === user.id) return;
+
+    this.pointManager.removePoint(
+      targetUser.id,
+      targetUser.username || targetUser.id,
+      1
+    );
+
+    console.log(
+      `Removed 1 point from ${targetUser.username} (${targetUser.id}) by ${user.username}`
+    );
+  }
+
+  private async showRanking(interaction: CommandInteraction) {
+    const limit = (interaction.options.get("limit")?.value as number) || 10;
+    const ranking = this.pointManager.getRanking(limit);
+
+    if (ranking.length === 0) {
+      await interaction.reply({
+        content: "まだポイントが記録されていません。",
+        ephemeral: true,
+      });
+      return;
+    }
+
+    const embed = new EmbedBuilder()
+      .setTitle("🏆 ポイントランキング")
```

Comment:
IMO: 以下の方がぱっと見わかりやすいかと
```suggestion
      .setTitle("🏆 Maximum Point ランキング")
```